packages/grafana-ui/src/components/Section/Section.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { SectionFill } from './SectionFill';
+import { SectionLabel } from './SectionLabel';
+
+/**
+ * @alpha
+ */
+export const Section = ({
```

Comment:
What's the purpose of this component? Does it only fit in the context of Segment? Is the label suppose to refer to something?

Reply:
It's a shortcut for creating horizontal sections / rows that may end with a filled space at the end (examples below). It can be used with any components but it's used with Segments at the moment. I'll move it `Segment` and rename to `SegmentsSections` to make it more explicit.

<img width="801" alt="Screen Shot 2021-08-16 at 12 25 02" src="https://user-images.githubusercontent.com/745532/129550012-43a5fc9c-8a04-4901-843c-897051d990ac.png">
<img width="800" alt="Screen Shot 2021-08-16 at 12 25 10" src="https://user-images.githubusercontent.com/745532/129550015-98e325eb-5787-44ae-b2e9-98d7c8ab1c41.png">
<img width="805" alt="Screen Shot 2021-08-16 at 12 25 17" src="https://user-images.githubusercontent.com/745532/129550017-6535c4d0-9c17-4c38-8810-ee279fef4906.png">

TODO:
- [ ] move Section to Segment

Reply:
We have `InlineField`, `inlineFieldRow` and `InlineLabel` components in grafana/ui, which should be able to achieve the same results. Or do they not work for this case?

Reply:
There a few differences:
- `SegmentSection` has fixed label styling (color and size) for consistency across editors
- `InlineField` takes only one child component, `SegmentSection` wraps multiple components

`SegmentSection` above vs `InflineField` below:

<img width="1017" alt="Screen Shot 2021-08-16 at 14 44 55" src="https://user-images.githubusercontent.com/745532/129565894-cebbdd74-596b-4bf9-aa41-9fdbeadc6b4c.png">

Reply:
What about wrapping multiple components into Fragment and passing them to `InlineLabel`? It just seems like these new components are 90% identical to the inline fields we already have, so I wonder if some customisation of the current component would make them work here?

Reply:
Make sense. I replaced them with `InlineFieldRow` and `InlineLabel` in 53901b0a0738aed163700fbfe85e675ccf0bc594. I hope it looks better now.